ansible/playbook.yaml

```
@@ -0,0 +1,98 @@
+- name: Update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+
+  vars_prompt: 
+
+  - name: trello_api_key
+    prompt: What is your Trello API Key?
+
+  - name: trello_api_token
+    prompt: What is your Trello API Token?
+
+  - name: trello_board_id
+    prompt: What is your Trello Board ID?
+
+
+  tasks:
+  - name: Install the latest version of Git
+    become: yes 
+    ansible.builtin.yum:
+      name: git
+      state: latest
+
+  - name: Install the latest version of Python
+    become: yes 
+    ansible.builtin.yum:
+      name: python311
+      state: latest    
```

Minor, but worth noting that you can combine these if you choose:
```suggestion
  - name: Install the latest versions of Git & Python
    become: yes 
    ansible.builtin.yum:
      name: 
        - git
        - python311
      state: latest 
```